README.md

```diff
@@ -1,13 +1,18 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project is about working with backend and creating an API.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+To create this API I had to work with NodeJs and express. I learned how to create API endpoints and how to return proper responses. When I got stucked with ideas or soultions I used Google and StackOverflow.
+
+##  Endpoints:
+- Root: `/`
+- books (shows the books Library): `/books` 
+- title: `/books?title=(book title)`
+- author: `/books?author=(book author)`
+- books by ID: `/books/id`
```

Comment:
Nice with some documentation! [Here](https://www.npmjs.com/package/express-list-endpoints)'s a cool lib that you can try out next time if you'd like to include the documentation in your API.